examples/with-nextjs-and-clerk-auth/src/components/DashboardCard.tsx

```diff
@@ -5,10 +5,11 @@ import { Card, CardHeader, CardContent, SvgIconProps } from '@mui/material';
 type IconVariant = 'info' | 'success' | 'critical';
 
 interface DashboardCardProps {
-  title: string;
+  title: string | ReactNode;
```

Comment:
The ReactNode type in React already includes string as one of its possible types, as shown in your screenshot. According to the ReactNode definition:

<img width="663" alt="Screenshot 2024-11-22 at 12 22 49" src="https://github.com/user-attachments/assets/64964869-f9b1-4a3f-b483-3dd276cd51cc">

This means that ReactNode can already accept string values, so explicitly declaring title: string | ReactNode is redundant

Reply:
Thanks! Updated